src/db.ts

```diff
@@ -48,6 +48,7 @@ const createSQLStatements = [
           id INTEGER PRIMARY KEY autoincrement,
           repo_id INTEGER NOT NULL,
           branch TEXT NOT NULL,
+          upstream_authors TEXT NOT NULL,
```

Comment:
This won't work on an existing deployment since the `pending_prs` table already exists. Instead, you can use `ALTER TABLE` after this `CREATE TABLE` statement instead.

For example (untested):

```sql
ALTER TABLE pending_prs
ADD COLUMN upstream_authors TEXT NOT NULL
DEFAULT "not-applicable";
```

Reply:
Oops. That makes sense. Sorry SQL noob over here... 🙂

Reply:
This one was painful. Apparently SQLite can't do `ALTER TABLE` conditionally. I added some code that I _think_ should work (at least, it worked to pass the tests...). Please take a look--I'll see about adding a test for it to make sure the column at least gets added if it wasn't there...